test/sql/hnsw_insert_empty_table.sql

```diff
@@ -0,0 +1,11 @@
+CREATE EXTENSION IF NOT EXISTS vector;
+CREATE EXTENSION IF NOT EXISTS lanterndb;
+
+CREATE TABLE small_world (
+    id varchar(3),
+    vector vector(3)
```

Comment:
Can you make this test with arrays and drop vector dependency?
And, use the small_world definition from `test_helpers` (just `\ir test_helpers/small_world_arrays.sql`)